kvmagent/kvmagent/test/vmmeta_testsuite/local/test_file_metadata_handler.py

```
@@ -0,0 +1,467 @@
+import json
+import os
+import shutil
+import tempfile
+import threading
+import time
+from unittest import TestCase
+
+from kvmagent.test.utils import pytest_utils
+from kvmagent.test.utils.stub import *
```

Ruff (0.15.7)

[error] `from kvmagent.test.utils.stub import *` used; unable to detect undefined names

(F403)